test/hotspot/jtreg/compiler/vectorapi/TestVectorMaskTrueCount.java

```diff
@@ -58,11 +59,11 @@ public class TestVectorMaskTrueCount {
         }
     }
 
-    static int maskAndTrueCountFloat(boolean[] a, boolean[] b, int idx) {
+    static int maskAndTrueCount(boolean[] a, boolean[] b, int idx, int SPECIES_length) {
```

Comment:
`SPECIES_length` confuses me. The first confusing thing is `SPECIES` in capital letters. The second is the name itself. From the code I see it has meaning of `count`. Maybe using `count` instead would be better?

Reply:
Thanks, I have modified the name of this parameter.